src/Connector.SqlServer/Utils/SqlTableName.cs

```diff
@@ -13,14 +13,14 @@ public sealed class SqlTableName
         /// </summary>
         public static SqlTableName FromUnsafeName(string rawTableName, SqlName schema)
         {
-            return new SqlTableName(SqlName.FromUnsafe(rawTableName), schema);
+            return new SqlTableName(SqlName.FromUnsafe(rawTableName), schema); 
```

Comment:
@zvirja 
here and on line 23 we use different syntax for method overloads. Can I make them alike?